### PR TITLE
Documentation - update architecture image

### PR DESCRIPTION
**Pull request by btmonier**
## Description

This PR updates the SVG image of the PHGv2 architecture diagram to its most current version

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Update PHGv2 architecture diagram
```

### CODE REVIEW

**Comment by lynnjo** [comment]
Do we want to update this diagram again?  Or is that what Cinta is looking to do?  On my copy, Ed crossed out the "GATK HaplotypeCaller" box in the Rare Allele Discovery section.  And I think Cinta and Bethany had opinions on what this pipeline should include

File: README.md [hunk 1]
```diff
@@ -172,4 +172,4 @@ More commonly used terms can be found [here](docs/terminology.md).
 * [hVCF format specifications](docs/hvcf_specifications.md)
 * [haplotype region handling](docs/hvcf_region_handling.md)
 * [PHGv2 terminology](docs/terminology.md)
-* [PHGv2 architecture](docs/img/architecture/phg_v2_architecture-Diagram.drawio.svg)
+  * [PHGv2 architecture](docs/img/architecture/phg_v2_architecture_20240411.svg)
```
Comment (tcasstevens) [inline README.md:175]: There's a space there that my OCD doesn't like ;)

**Review by tcasstevens — APPROVED** [review]

**Review by lynnjo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (btmonier) [reply README.md]: Fixed - I do not know how this happened. I will blame Intellij...

Comment (tcasstevens) [reply README.md]: Thanks Brandon!

**Pull request merged**